src/main/java/zmaster587/advancedRocketry/dimension/DimensionProperties.java

```diff
@@ -137,17 +138,17 @@ public static AtmosphereTypes getAtmosphereTypeFromValue(int value) {
 	 */
 	public static final ResourceLocation atmosphere = new ResourceLocation("advancedrocketry:textures/planets/Atmosphere2.png");
 	public static final ResourceLocation atmosphereLEO = new ResourceLocation("advancedrocketry:textures/planets/AtmosphereLEO.png");
-	public static final ResourceLocation atmGlow = new ResourceLocation("advancedrocketry:textures/planets/atmGlow.png");
-	public static final ResourceLocation planetRings = new ResourceLocation("advancedrocketry:textures/planets/rings.png");
-	public static final ResourceLocation planetRingShadow = new ResourceLocation("advancedrocketry:textures/planets/ringShadow.png");
-	public static final ResourceLocation shadow = new ResourceLocation("advancedrocketry:textures/planets/shadow.png");
-	public static final ResourceLocation shadow3 = new ResourceLocation("advancedrocketry:textures/planets/shadow3.png");
+	public static final ResourceLocation atmGlow = new ResourceLocation("advancedrocketry:textures/planets/AtmGlow.png");
+	public static final ResourceLocation planetRings = new ResourceLocation("advancedrocketry:textures/planets/Rings.png");
+	public static final ResourceLocation planetRingShadow = new ResourceLocation("advancedrocketry:textures/planets/RingsShadow.png");
+	public static final ResourceLocation shadow = new ResourceLocation("advancedrocketry:textures/planets/Shadow.png");
+	public static final ResourceLocation shadow3 = new ResourceLocation("advancedrocketry:textures/planets/Shadow3.png");
 
 	public static enum PlanetIcons {
-		EARTHLIKE(new ResourceLocation("advancedrocketry:textures/planets/Earthlike.png")),
+		EARTHLIKE(new ResourceLocation("advancedrocketry:textures/planets/EarthLike.png")),
```

Comment:
Standard should be all lowercase for any `ResourceLocation`, 1.8 and later typically enforces this by just running `ToLower()` on any string you pass it, having caps here could render 1.7 and 1.12 more divergent than they already are.

```suggestion
		EARTHLIKE(new ResourceLocation("advancedrocketry:textures/planets/earthlike.png")),
```